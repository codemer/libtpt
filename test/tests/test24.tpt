@# HASH Test
@set(op.key, "value")\
${op.key}
@set(op.array, "apple", "pear", "banana", "pineapple", "orange", "strawberry")\
@foreach object (op.array) {
${object}
}
