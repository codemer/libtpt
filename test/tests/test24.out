value
apple
pear
banana
pineapple
orange
strawberry
